src/style.css

```diff
@@ -1,3 +1,88 @@
+@import 'https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap';
```

Comment:
According to the checklist, you should remember to use fallback fonts - alternative font-family in case the main one doesn't work. You've specified 'Roboto' from Google Fonts, but it's a good practice to specify a generic font family as a fallback. For instance, if 'Roboto' fails to load for some reason, the browser can use 'Arial' or a generic 'sans-serif' font. You can do this by adding fallback fonts to the --roboto-font variable in the root. For example: 'Roboto', Arial, sans-serif.